db/migrate/20170622210725_create_standard_statements.rb

```diff
@@ -0,0 +1,19 @@
+class CreateStandardStatements < ActiveRecord::Migration
+  def change
+    create_table :standard_statements do |t|
+      t.string :uri
+      t.string :doc
+      t.string :statement_notation
+      t.string :statement_label
+      t.string :description
```

Comment:
Should this be `t.text` instead for the description?  I believe `t.string` is limited to 255 characters..